pkgs/top-level/all-packages.nix

```
@@ -2417,6 +2417,8 @@ in
 
   evdevremapkeys = callPackage ../tools/inputmethods/evdevremapkeys { };
 
+  geebar-libinput = callPackage ../tools/inputmethods/gebaar-libinput { };
```

See my previous comment

```suggestion
  geebar-libinput = callPackage ../tools/inputmethods/gebaar-libinput {
    stdenv = gcc8Stdenv;
  };
```